hat/conf/play.conf

```diff
@@ -82,8 +82,18 @@ play {
       allowed = [".hubofallthings.net", ".hubat.net", ${?HAT_DOMAIN}]
     }
 
-    enabled = []
+    enabled = [
+        "com.github.stijndehaes.playprometheusfilters.filters.StatusCounterFilter",
+        "com.github.stijndehaes.playprometheusfilters.filters.StatusAndRouteLatencyFilter",
+        "org.hatdex.hat.utils.LoggingFilter",
+        "play.filters.hosts.AllowedHostsFilter",
+        "play.filters.cors.CORSFilter",
+        "play.filters.gzip.GzipFilter",
+        "org.hatdex.hat.utils.TLSFilter"
+    ]
+
     //    enabled += "play.filters.headers.SecurityHeadersFilter"
+
     enabled += "play.filters.hosts.AllowedHostsFilter"
```

Comment:
all these three filters (line 97-99) are applied twice... not very efficient

Reply:
Fixed